### PR TITLE
Fix for declaredValue in Transfer Details

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#13818

*Description of changes:*
- Submit Home Declared Value as a _number_ not a _string_

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by ChasBoucher — APPROVED** [review]
Looks good.

File: ppr-ui/src/components/mhrTransfers/TransferDetails.vue [hunk 1]
```diff
@@ -148,9 +148,9 @@ export default defineComponent({
         customRules(maxLength(80), props.validateTransferDetails ? required('Enter consideration') : [])
     )
 
-    const updateCompensation = (declaredValue: number) => {
+    const updateCompensation = (declaredValue: any) => {
       // copy Declared Value into Consideration field - the initial time only
-      if (!localState.consideration && localState.declaredValue) {
+      if (!localState.consideration && localState.declaredValue && parseInt(declaredValue)) {
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/TransferDetails.vue]: If `parseInt` is required, wouldn't `declaredValue` be of type `string`?

File: ppr-ui/src/components/mhrTransfers/TransferDetails.vue [hunk 2]
```diff
@@ -167,21 +167,21 @@ export default defineComponent({
 
     watch(
       () => localState.declaredValue,
-      async (val: boolean) => {
-        await setMhrTransferDeclaredValue(val)
+      async (val: any) => {
+        await setMhrTransferDeclaredValue(parseInt(val) ? parseInt(val) : null)
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/TransferDetails.vue:171]: Same as above

**Review by cameron-eyds — APPROVED** [review]
Looks good, just some minor comments

File: ppr-ui/src/components/mhrTransfers/TransferDetails.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrTransfers/TransferDetails.vue]: Updated!

File: ppr-ui/src/components/mhrTransfers/TransferDetails.vue (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrTransfers/TransferDetails.vue:171]: Updated!

**Comment by dimak1** [comment]
@cameron-freshworks thanks, good catch. Updated.

**Review by doug-lovett — APPROVED** [review]

**Pull request merged**